examples/run-blockchain/index.js

```diff
@@ -17,14 +17,24 @@ var blockchainDB = levelMem()
 
 var state = new Trie()
 
-var blockchain = new Blockchain(blockchainDB)
-blockchain.ethash.cacheDB = level('./.cachedb')
+var validate = false
+
+var hardfork = 'byzantium'
+
+var blockchain = new Blockchain({ db: blockchainDB, hardfork, validate })
+
+if (validate) {
```

Comment:
If we remove the `validate` variable, this if should also be removed.

Reply:
done